<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelemetryWindow</class>
 <widget class="QMainWindow" name="TelemetryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Telemetria</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Bateria</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QProgressBar" name="progressBarBattery">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Kąt</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QGroupBox" name="groupBox_4">
             <property name="title">
              <string>Zadany</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <widget class="QProgressBar" name="progressBarTargetAngle">
                <property name="minimum">
                 <number>-12000</number>
                </property>
                <property name="maximum">
                 <number>12000</number>
                </property>
                <property name="value">
                 <number>0</number>
                </property>
                <property name="format">
                 <string>%v m°</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_5">
             <property name="title">
              <string>Aktualny</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <widget class="QProgressBar" name="progressBarAngle">
                <property name="minimum">
                 <number>-12000</number>
                </property>
                <property name="maximum">
                 <number>12000</number>
                </property>
                <property name="value">
                 <number>0</number>
                </property>
                <property name="format">
                 <string>%v m°</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_9">
           <property name="title">
            <string>Różnica</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QProgressBar" name="progressBarDiffAngle">
              <property name="minimum">
               <number>-12000</number>
              </property>
              <property name="maximum">
               <number>12000</number>
              </property>
              <property name="value">
               <number>0</number>
              </property>
              <property name="format">
               <string>%v m°</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Prędkość</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QGroupBox" name="groupBox_6">
             <property name="title">
              <string>Zadana</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_7">
              <item row="0" column="0">
               <widget class="QProgressBar" name="progressBarTargetSpeed">
                <property name="minimum">
                 <number>-400000</number>
                </property>
                <property name="maximum">
                 <number>400000</number>
                </property>
                <property name="value">
                 <number>0</number>
                </property>
                <property name="format">
                 <string>%v mrpm</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_7">
             <property name="title">
              <string>Aktualna</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="0" column="0">
               <widget class="QProgressBar" name="progressBarSpeed">
                <property name="minimum">
                 <number>-400000</number>
                </property>
                <property name="maximum">
                 <number>400000</number>
                </property>
                <property name="value">
                 <number>0</number>
                </property>
                <property name="format">
                 <string>%v mrpm</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_10">
           <property name="title">
            <string>Różnica</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <widget class="QProgressBar" name="progressBarDiffSpeed">
              <property name="minimum">
               <number>-400000</number>
              </property>
              <property name="maximum">
               <number>400000</number>
              </property>
              <property name="value">
               <number>0</number>
              </property>
              <property name="format">
               <string>%v mrpm</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_8">
        <property name="title">
         <string>Zapis do pliku</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_9">
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButtonStartDump">
           <property name="text">
            <string>Rozpocznij</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButtonStopDump">
           <property name="text">
            <string>Zakończ</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonReturn">
        <property name="text">
         <string>Powrót</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
